pkg/extender/extender_test.go

```diff
@@ -372,7 +395,19 @@ func TestExtender(t *testing.T) {
 	t.Run("kubevirtPodSchedulingNonHyperconvergence", kubevirtPodSchedulingNonHyperconvergence)
 	t.Run("nodeMarkedUnschedulableTest", nodeMarkedUnschedulableTest)
 	t.Run("nodeMarkedUnschedulableWFFCVolTest", nodeMarkedUnschedulableWFFCVolTest)
-	t.Run("teardown", teardown)
+	t.Run("teardown", func(t *testing.T) {
+		teardown(t, false)
+	})
+}
+
+func TestExtenderWithCache(t *testing.T) {
+	t.Run("setup", func(t *testing.T) {
+		setup(t, true)
+	})
+	t.Run("multipleVolumeTestWithCache", multipleVolumeTestWithCache)
```

Comment:
As far as UTs are concerned, we can run all above UTs with cache enabled too right? Let's do it then.

Reply:
Since an UT with mocked informer cache for GetPVC is enough, I feel no need to mock it in all existing tests.

Reply:
Is the `multipleVolumeTestWithCache` dependent on `setCacheInstance` test to configure the cache test instance here? If that's the case, running it alone would fail, right?

Reply:
yes, running alone that will fail.